input/index.cshtml

```diff
@@ -12,14 +12,14 @@ NoGutter: true
           Just a few classes, so easily embeddable in your own code, or a very small assembly to ship with your code.<br/>
           The code is highly portable, which in general makes it easy to adapt for new platforms.</i>
         </p>
-        <p>
-            <a class="btn btn-primary btn-lg" href="/docs/quickstart" role="button">Quickstart &raquo;</a>
-        </p>
```

Comment:
IDE complains about anchor using a button role here. The change doesn't make any difference functionality wise